main_test.go

```diff
@@ -544,6 +605,49 @@ func TestGetVMName(t *testing.T) {
 	}
 }
 
+func TestCheckIPv6Capable(t *testing.T) {
+	tests := []struct {
+		desc        string
+		httpHandler func(http.ResponseWriter, *http.Request)
+		wantOutput  bool
+	}{
+		{
+			desc: "case with check on ipv6 enabled host",
```

Comment:
See: go/go-style/decisions#subtest-names

I'm thinking `v6 enabled` and `v6 not enabled`.